.circleci/config.yml

```diff
@@ -144,3 +144,7 @@ workflows:
             branches:
               only:
                 - master
+          requires:
+            - Lint
+            - Build
+            - Test
```

Comment:
I think we won't be notified on Slack if either of these steps fails on the `master` branch, `- notify_slack_on_failure` currently runs only for the "release to GitHub" job

Reply:
You're completely right! I included the `notification_slack_on_failure` for all steps, but only when merged to `master` to avoid noise in the Slack channel.